models/Recording.js

```diff
@@ -534,44 +517,77 @@ module.exports = function(sequelize, DataTypes) {
         ]
       },
       order: order,
-      include: [
-        {
-          model: models.Group,
-          attributes: ["groupname"]
-        },
-        {
-          model: models.Tag,
-          attributes: ["what", "detail", "automatic", "taggerId", "confidence"],
-          required: false
-        },
-        {
-          model: models.Track,
-          where: {
-            archivedAt: null
-          },
-          attributes: ["id"],
-          required: false,
-          include: [
-            {
-              model: models.TrackTag,
-              attributes: ["what", "automatic", "UserId", "confidence"],
-              required: false
-            }
-          ]
-        },
-        {
-          model: models.Device,
-          attributes: ["id", "devicename"]
-        }
-      ],
+      include: getRecordingInclude(),
       limit: limit,
       offset: offset,
       attributes: Recording.queryGetAttributes
     };
-
     return this;
   };
 
+  function getRecordingInclude() {
+    return [
+      {
+        model: models.Group,
+        attributes: ["groupname"]
+      },
+      {
+        model: models.Tag,
+        attributes: models.Tag.userGetAttributes,
+        include: [
+          {
+            association: "tagger",
+            attributes: ["username", "id"]
+          }
+        ]
+      },
+      {
+        model: models.Track,
+        where: {
+          archivedAt: null
+        },
+        attributes: [
+          "id",
+          [
+            Sequelize.fn(
+              "json_build_object",
+              "start_s",
+              Sequelize.literal(`"Tracks"."data"#>'{start_s}'`),
+              "end_s",
+              Sequelize.literal(`"Tracks"."data"#>'{end_s}'`)
+            ),
+            "data"
+          ]
```

Comment:
What does it mean when you put an array in the `attributes` array like this?

Reply:
['test_data', 'test'] e.g. which would translate too "select test_data as test"